docs/usage.md

```diff
@@ -0,0 +1,60 @@
+# ESI Usage
+
+This document highlights OpenStack CLI commands used to peform common ESI functions. A complete listing of commands can be found by reading the [complete OpenStack CLI documentation](https://docs.openstack.org/python-openstackclient/latest/cli/command-list.html). Each entry here also links to the appropriate entry in the OpenStack CLI documentation.
+
+* [Identity](#identity)
+* [Isolation](#isolation)
+* [Maintenance](#maintenance)
+* [Provisioning](#provisioning)
+
+## <a name="identity"></a>Identity
+
+### Administrator
+
+* **Create project:** *openstack project create \<project-name\>* [[full reference](https://docs.openstack.org/python-openstackclient/latest/cli/command-objects/project.html#project-create)]
```

Comment:
Optional suggestion, but personally I think commands looks better with backticks instead of italics, and you don't have to escape the angle brackets.
For instance: `openstack project create <project-name>`
and in case of multiple commands, you can have a block that starts and ends with 3 backticks.
```
openstack project create <project-name>
openstack user create --project <project-name> --password <password> <user-name>
```

Reply:
You're definitely right - updated!